test/sanity/pep8/legacy-files.txt

```diff
@@ -1,3 +1,8 @@
+hacking/conf2yaml.py
+lib/ansible/constants.py
+lib/ansible/cli/config.py
+lib/ansible/config/data.py
+lib/ansible/config/manager.py
```

Comment:
the merge was too hard to do once the files were split, since the changes were pure style i just ignored them.